json-schemas/v3/tech-record/get/car/complete/index.json

```diff
@@ -237,26 +237,23 @@
 			"type": "string"
 		},
 		"techRecord_vehicleSubclass": {
+			"title": "Vehicle Subclass",

```

Comment:
Files under `json-schemas/` are generated by a script. This essentially "compiles" the schemas inside of the `json-definitions/` folder to one big schema. I believe this is done by the [json-schema-deref-sync]( https://www.npmjs.com/package/json-schema-deref-sync) package. This means you can write re-useable pieces of json schema and use them in many places, but only use big de-refed schemas at runtime. I _think_ it's because you have to use de-refed ones at runtime.